apis/python/tests/test_tiledbobject.py

```diff
@@ -0,0 +1,42 @@
+import tiledbsoma as soma
+
+# Checking that objects _do_ exist is already done (thoroughly) in other
+# tests. Here we check the negative cases.
+#
+# Specifically, we're checking these don't raise, and all return False.
+
+
+def test_dataframe_exists():
+    assert not soma.DataFrame.exists("/nonesuch/no/nope/never/ever")
+    assert not soma.DataFrame.exists("foo://bar")
+    assert not soma.DataFrame.exists("s3://@@@@ILLEGAL@@@@")
```

Comment:
At the risk of belaboring the unit test aspect of this PR, all of these tests could be shortened with `@pytest.mark.parametrize`

Reply:
done